superset/databases/utils.py

```diff
@@ -55,7 +55,7 @@ def get_indexes_metadata(
 
 def get_col_type(col: dict[Any, Any]) -> str:
     try:
-        dtype = f"{col['type']}"
+        dtype = f"{[col['type']]}"[1:-1]
```

Comment:
Hard for me to make sense of this line of code before and after the change. Maybe needs a comment, docstring and/or better typing?

Also if this is solving a trino-specific issue, does it belong somewhere in db_engine_spec?